public/docs/ts/latest/guide/template-syntax.jade

```diff
@@ -542,7 +542,7 @@ table
 // #docregion property-binding-7
 :marked
   ### Binding target
-  a id between enclosing square brackets identifies the target property. The target property in the following code is the image element’s `src` property.
+  An element property between enclosing square brackets identifies the target property. The target property in the following code is the image element’s `src` property.
```

Comment:
Yeah, is a confusing point. I still think that what you see in the HTML is an attribute and then it becomes a property when it gets processed by the browser. So it is an attribute with square brackets.

Or perhaps I am wrong. Let's get another opinion in here.